org.sqlitebrowser.sqlitebrowser.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.sqlitebrowser.sqlitebrowser
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: sqlitebrowser
+rename-icon: sqlitebrowser
+rename-desktop-file: sqlitebrowser.desktop
+rename-appdata-file: sqlitebrowser.desktop.appdata.xml
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home
+  - --share=network
```

Comment:
does it really need network access?

Reply:
Every Qt application has this line where it says network failed in stdout, but I don't know what it does.

Reply:
There's support for pushing to [DBHub](https://dbhub.io/).  
Open a db and look in the `Remote` tab. You might need to enable the remote feature through settings.

Reply:
Added it back.

Reply:
ah ok.